test/xpu_api/random/interface_tests/distributions_methods.pass.cpp

```diff
@@ -296,10 +303,41 @@ test(sycl::queue& queue)
 
     // Random number generation
     {
+        {
+            Distr _d1(0.1);
+            Distr _d2(0.1);
+            if (_d1 == _d2)
+            {
+
+            }
+
+            std::ostringstream out;
+            std::istringstream in("0.5");
+
+            in >> _d1;
+            assert(check_params(_d1));
+            out << _d1;
+            assert(check_output);
+
+            if (_d1 != _d2)
+            {
+
+            }
```

Comment:
Is it extra code or it will be extended later in this PR?

Reply:
Overall I don't know exactly where distributions should be compared in the code